submissions/misshatsker/oop-exercise/index.js

```diff
@@ -0,0 +1,62 @@
+class MyInhabitant {
+   static inhabitantProperties = ['species', 'name', 'gender', 'saying'];
+   static delimiter = '; ';
+
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+      this.properties = [...MyInhabitant.inhabitantProperties];
+   }
+
+   getInhabitantInfo() {
+      const values = this.properties.map((key) => this[key]);
+      return values.join(MyInhabitant.delimiter);
```

Comment:
(1) poor naming - too generic variable names. General requirements with regrads to the code style are still valid and remain valid after the course completed for all your professional life.
(2) why not to chain `Array` methods?